css/pages/review.scss

```diff
@@ -0,0 +1,88 @@
+// Review
+.page {}
+
+.page--review {
+  color: #404040;
+
+  .page--review__title {
+    text-align: center;
+    font-style: bold;
+    font-weight: 700;
+    font-size: 2.5em;
+    letter-spacing: 3px;
+    margin: 40px;
+  }
+
+  @media (max-width: $s__breakpoint--small) {
+    .page--review__title {
+      margin-top: 1000px;
```

Comment:
it was so i could see the changes i made when i resized (otherwise they were covered by Peter's nav bar)